t3/runners/rmg_runner.py

```diff
@@ -142,7 +147,13 @@ def rmg_job_converged(project_directory: str) -> bool:
                 if 'MODEL GENERATION COMPLETED' in lines[len_lines - 1 - i]:
                     rmg_converged = True
                     break
-    return rmg_converged
+    if not rmg_converged and os.path.isfile(rmg_err_path):
+        with open(rmg_err_path, 'r') as f:
+            lines = f.readlines()
+        for line in lines:
```

Comment:
It's not really important but I guess for saving a fraction of a millisecond, we could run through the lines in reverse as 'Error' will always appear last in the file. Or instead of looking for the keyword Error, we could look for 'Traceback'
Again though, not important, just a thought